packages/netskope/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.1.2"
+  changes:
+    - description: Updated the check in boolean fields
+      type: enhancement
```

I suggest giving users more details.

```suggestion
    - description: Fix boolean conversion logic to accept "true", "false", "yes", and "no" as strings. Correct the type of `is_alert` and `is_web_universal_connector` to boolean.
      type: bugfix
```